package.json

```diff
@@ -66,16 +66,23 @@
     "@size-limit/file": "^8.0.0",
     "@size-limit/webpack": "^8.0.0",
     "@stylelint/postcss-css-in-js": "^0.38.0",
+    "@testing-library/cypress": "^8.0.0",
+    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/react-hooks": "^7.0.0",
+    "@testing-library/user-event": "^13.1.1",
     "@types/dedent": "^0.7.0",
     "@types/dotenv-safe": "^8.1.2",
     "@types/lodash": "^4.14.194",
     "@types/prettier": "^2.7.2",
     "@types/react": "^17.0.26",
     "@types/react-dom": "^17.0.9",
+    "@types/testing-library__jest-dom": "^5.14.5",
     "@typescript-eslint/eslint-plugin": "^5.57.1",
     "@typescript-eslint/parser": "^5.57.1",
     "babel-jest": "^28.1.3",
     "browserslist": "^4.16.6",
+    "classnames": "^2.3.2",
```

Comment:
insted of classnames please use `clsx`. And remove it from root, as it's planned to be used only inside components package.

Reply:
Changed to `clsx` but if I remove from toor devDeps, I keep getting the [eslint import/no-extraneous-dependencies error](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-extraneous-dependencies.md). Doesn't it make sense to keep as devDep on root?

Reply:
I think it should be moved to its package where it will be used as a normal dependency, not a dev, as it will be executed in runtime

Reply:
Right, but I have it on orbit-components as dependency and it still complains 😞